packages/solana/package.json

```diff
@@ -19,6 +19,11 @@
       "import": "./dist/esm/exports/react.js",
       "default": "./dist/esm/exports/react.js"
     },
+    "./vue": {
+      "types": "./dist/types/exports/vue.d.ts",
+      "import": "./dist/esm/exports/vue.js",
+      "default": "./dist/esm/exports/vue.js"
+    },
```

Comment:
the `/vue` subpath was not exported. I wonder how this ever worked. The build failed without this.

Reply:
I believe this was recently merged by @phoenixVS

Reply:
Great. I'll rebase accordingly